SpywareFilter/sections/tracking_servers.txt

```diff
@@ -42,6 +42,9 @@
 ||visilabs.com^$third-party
 ||visilabs.net^$third-party
 !
+||tracking.adweb.co.kr^$third-party
+||tracking02.adweb.co.kr^$third-party
+||wcs.naver.com^$image
```

Comment:
better move to `specific`. In this file only one modified is allowed - `$third-party`

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/76128/commits/4ddb981954c921035fc4d333b5fe9309e27ae287